sources/mongodb/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""Source that loads collections form any a mongo database, supports incremental loads."""
+
+from typing import Any, Iterable, List, Optional
+
+import dlt
+from dlt.extract.source import DltResource
+
+from .helpers import (
+    MongoDbCollectionConfiguration,
+    MongoDbCollectionResourceConfiguration,
+    client_from_credentials,
+    collection_documents,
+)
+
+
+@dlt.source
+def mongodb(
+    connection_url: str = dlt.secrets.value,
```

Comment:
what if database is part of `connection_url`? do we still need `database` as string? if so we could make optional

Reply:
Just gave a look at the [docs](https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html) to confirm, apparently you cannot pass database with the connection string.